.github/CONTRIBUTING.md

```diff
@@ -141,6 +141,31 @@ var compare = (a + b * c) != (d - e * f);
 
       `camelCaseWord`
 
+  15. Simple Branches:
+
+```C#
+if (condition) {
+    ...
+} else if (condition) {
+    ...
+} else {
+    ...
+}
+```
+
+  16. Switches:
+
+```C#
+switch (value) {
+    case 1:
+        ...
+        break;
+    default:
+        ...
+        break;
+}
+```
```

Comment:
I worry that even this is too much to say on switches if I'm honest - I don't want to stop people using the single line syntax, e.g.:

```C#
switch (value) {
    case 1:  return 2;
    case 2:  return 1;
    default: return 0;
}
```